modules/aws-limits.adoc

```diff
@@ -63,7 +63,7 @@ To use the `us-east-1` region, you must increase the EIP limit for your account.
 |350 per region
 |The default installation creates 21 ENIs and an ENI for each availability zone in your region. For example, the `us-east-1` region contains six availability zones, so a cluster that is deployed in that zone uses 27 ENIs. Review the link:https://aws.amazon.com/about-aws/global-infrastructure/[AWS region map] to determine how many availability zones are in each region.
 
-Additional ENIs are created for additional machines and elastic load balancers that are created by cluster usage and deployed workloads.
+Additional ENIs are created for additional machines and ELB load balancers that are created by cluster usage and deployed workloads.
```

Comment:
@dfitzmau I would consider dropping the "ELB" here. Earlier in the topic, it's already been introduced that this information applies to ELB resources. The acronym seems unnecessary.